providers/base/units/rt-performance-tests/jobs.pxu

```diff
@@ -0,0 +1,11 @@
+id: rt-performance-tests/cyclictest_rt_latency
+category_id: rt-performance-tests
+_summary:
+    Run cyclictest to test scheduling latency. Default run time is 24 hours.
+plugin: shell
+requires:
+    package.name == 'rt-tests'
```

Comment:
Where does this package come from?

Reply:
In `checkbox-iiotg`, we package it into snap. Do you think if we could package it into checkbox snap too?

Reply:
Yes, both for the snap and the Debian package.

So please:

1. add the `rt-tests` package to the list of `stage-packages` for the `checkbox-provider-base` (for [series22](https://github.com/canonical/checkbox/blob/main/checkbox-core-snap/series22/snap/snapcraft.yaml#L276) at least, not sure if this is required for older series)
2. add a packaging meta data unit (we usually name them `packaging.pxu`) in `rt-performance-tests` to create the Debian dependency (you can see an example [here](https://github.com/canonical/checkbox/blob/main/providers/base/units/camera/packaging.pxu))

Ping me if you need help.

Reply:
@pieqq Done. This is my first time editing packaging.pxu. Please take a look.